services/ops/TopicWatcher/tests/test_topic_watcher.py

```diff
@@ -298,10 +298,14 @@ def test_watch_topic_new_group(volttron_instance, agent, cleanup_db):
     gevent.sleep(6)
 
     assert len(alert_messages) == 2
+
     assert "Topic(s) not published within time limit: ['fakedevice', " \
            "'fakedevice2/all', ('fakedevice2/all', 'point')]" in alert_messages or \
            "Topic(s) not published within time limit: ['fakedevice', " \
-           "('fakedevice2/all', 'point')], 'fakedevice2/all'" in alert_messages
+           "('fakedevice2/all', 'point')], 'fakedevice2/all'" in alert_messages or \
+           "Topic(s) not published within time limit: ['fakedevice', ('fakedevice2/all', 'point'), " \
```

Comment:
The check you added is the right one. the one above does not have the list right.  this test case only has different order of 
[ 'fakedevice', 'fakedevice2/all' , (fakedevice2/all, point) ] because order is not fixed. But it should be one list.